chainer/training/extensions/log_report.py

```diff
@@ -80,6 +80,7 @@ def __call__(self, trainer):
 
             updater = trainer.updater
             stats_cpu['epoch'] = updater.epoch
+            stats_cpu['epoch_detail'] = updater.epoch_detail
```

Comment:
Then, could you remove it? Also, could you send it as an another PR if you think it is worth doing so.